czo2hs.py

```diff
@@ -157,17 +158,17 @@ def migrate_czo_row(czo_row_dict, row_no=1, progress_report=5):
                                                                        error_item["error_msg_list"]])))
     logging.info(text_emphasis("CZO_ID <---> HS_ID Lookup Table"))
     logging.info(czo_hs_id_lookup_df.to_string())
-    results_file = os.path.join(logdir, 'czo_hs_id_{}.csv'.format(start_time))
-    logging.info("*" * 20 + "Saving Lookup Table to {}".format(results_file) + "*" * 20)
+    czo_hs_id_lookup_csv_path = os.path.join(logdir, 'czo_hs_id_{}.csv'.format(start_time_str))
+    logging.info(text_emphasis("Saving Lookup Table to {}".format(czo_hs_id_lookup_csv_path)))
 
-    czo_hs_id_lookup_df.to_csv(results_file, encoding='utf-8', index=False)
+    czo_hs_id_lookup_df.to_csv(czo_hs_id_lookup_csv_path, encoding='utf-8', index=False)
 
     # upload log file and results file to hydroshare
-    logging.info("*"*20 + "Uploading log file to HS" + "*"*20)
+    logging.info(text_emphasis("Uploading log file to HS"))
     hs = CZO_HS_Account_Obj.get_hs_by_czo("default")
     hs_id = hs.createResource("CompositeResource",
-                              "czo2hs migration log files {}".format(start_time),
+                              "czo2hs migration log files {}".format(start_time_str),
                               )
-    file_id = hs.addResourceFile(hs_id, os.path.join(logdir, log_file_name))
-    file_id = hs.addResourceFile(hs_id, results_file)
+    file_id = hs.addResourceFile(hs_id, log_file_path)
+    file_id = hs.addResourceFile(hs_id, czo_hs_id_lookup_csv_path)
     print("Log files uploaded to HS res at {}".format(hs_id))
```

Comment:
let's move LOGDIR and make it a constant; move it to settings.py

Reply:
czo_hs_id_lookup_csv_path is too long. shorter, more concise variable names, please

Reply:
change name back to results_file

Reply:
done in new commit (not sure why github is not updating)